### PR TITLE
fix #6: add missing files for dating sim

### LINKED ISSUE #6: fix dating sim
:(

### PR DESCRIPTION
**Pull request by rawxfish**

### CODE REVIEW

File: docs/static/puzzle_resources/the_dating_sim/htaccess.txt [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+#AddOutputFilterByType DEFLATE application/wasm
+
+
+# Force browser to check if cache needs to be refreshed or not (304) for XHR
+# (preserve browser cache, but revalidate)
+<IfModule mod_headers.c>
+  Header set Cache-Control "max-age=0"
+</IfModule>
+
+# Disable ETags which don't play nice with mod_gzip and HTTP/1.1
```
Comment (akirabaruah) [inline docs/static/puzzle_resources/the_dating_sim/htaccess.txt:19]: Do we need this block of comments?

**Review by akirabaruah — APPROVED** [review]

File: docs/static/puzzle_resources/the_dating_sim/htaccess.txt (hunk 1 above)
Comment (rawxfish) [reply docs/static/puzzle_resources/the_dating_sim/htaccess.txt:19]: Almost all of the stuff in this directory was generated by renpy - I don't actually know what most of this does...

Comment (akirabaruah) [reply docs/static/puzzle_resources/the_dating_sim/htaccess.txt:19]: Ah gotcha. If we remove this file, does Dating Sim still work?

IIUC, this htaccess file configures Apache web servers, which might not apply here?...unless Renpy web uses it for its own configuration?

Seems like it doesn't hurt to keep it, but if it's not needed, we should probably remove it.

**Review by akirabaruah — APPROVED** [review]

File: docs/static/puzzle_resources/the_dating_sim/htaccess.txt (hunk 1 above)
Comment (rawxfish) [reply docs/static/puzzle_resources/the_dating_sim/htaccess.txt]: looks like it's not necessary, removed it.

**Pull request merged**